core/services/relay/evm/mercury/utils/feeds.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	"encoding/binary"
+
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+)
+
+type FeedVersion uint16
+
+const (
+	_         FeedVersion = 0 // reserved to prevent errors where a zero-default creeps through somewhere
+	REPORT_V1 FeedVersion = 1
+	REPORT_V2 FeedVersion = 2
+	REPORT_V3 FeedVersion = 3
```

Comment:
nit/ Could use iota?
```suggestion
	_         FeedVersion = iota // reserved to prevent errors where a zero-default creeps through somewhere
	REPORT_V1
	REPORT_V2
	REPORT_V3
```

Reply:
Done, lmk if it can be formatted better